docs/sources/tempo/operations/best-practices.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Best practices for traces
+menuTitle: Best practices
+description: Learn about the best practices for traces
+weight: 20
+---
+
+# Best practices for traces
+
+This page provides some general best practices for tracing.
+
+## Span and resource attributes
```

Comment:
If this is a best practices doc, should it contain the introductory material about spans and resource attributes? I understand why it's here -- because the naming attributes builds upon the span and resource content. This material could probably be reduced, though. I'm tempted to do that. What do you think, @joe-elliott ?

Reply:
I edited the content down and refocused it a bit.